client-app/core/api/graphql/organization/mutations/changeOrganizationLogo/index.ts

```diff
@@ -0,0 +1,14 @@
+import { graphqlClient } from "../../../client";
+import mutationDocument from "./changeOrganizationLogo.graphql";
```

Comment:
use type generator and import from types.ts

Reply:
fixed